### PR TITLE
feat: add support for Lighthouse Reports

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

**Comment by owenvoke** [comment]
I need to confirm that this works as expected. On an unrelated note, I'm going to work on adding PHP 8.1 / 8.2 testing to the package. 👍🏻

File: src/Resources/LighthouseReport.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OhDear\PhpSdk\Resources;
+
+class LighthouseReport extends ApiResource
+{
+    public int $id;
+
+    public int $performanceScore;
```
Comment (freekmurze) [inline src/Resources/LighthouseReport.php:9]: The score can also be `null` (this happens when some Lighthouse audits fail completely)

Comment (owenvoke) [reply src/Resources/LighthouseReport.php:9]: Is that the same for all scores? 👀

Comment (freekmurze) [reply src/Resources/LighthouseReport.php:9]: Yup!

Comment (owenvoke) [reply src/Resources/LighthouseReport.php]: Updated, used `?` nullable syntax as we have to support PHP 7.4. Is there a plan to update to PHP 8.1 and higher or something at some point?

Also, made the time ms values nullable, but not sure if they should be as well?

**Comment by freekmurze** [comment]
I would be ok with dropping support for PHP 8.0 and below, so the codebase can be modernised.

**Comment by owenvoke** [comment]
Great! Added in #33

**Comment by owenvoke** [comment]
This is now ready whenever you'd like to merge, @freekmurze 👍🏻

**Comment by freekmurze** [comment]
Thank you so much for your work on this!

**Pull request merged**